templates/core/about/badges.html

```diff
@@ -4,50 +4,51 @@
 
 {%- block body -%}
 	<h1>Badges</h1>
-
-	<div class="container about">
-	<p>
-		You can use badges to show state of your documentation to your users.
-		The default badge will be pointed at the latest version of a crate.
-		You can use <code>version</code> parameter to show status of documentation for
-		any version you want.
-	</p>
-
-	<p>
-		Badge will display in blue if docs.rs is successfully hosting your crate
-		documentation, and red if building documentation failing.
-	</p>
-
-	<p>Example badges for mio crate:</p>
-	<table class="pure-table pure-table-horizontal">
-		<thead>
-		<tr>
-			<th>URL</th>
-			<th>Badge</th>
-		</tr>
-		</thead>
-
-		<tbody>
-		{%- set mio_badge = "https://docs.rs/mio/badge.svg" -%}
-		<tr>
-			<td>Latest version: <a href="{{ mio_badge | safe }}">{{ mio_badge }}</a></td>
-			<td><img src="{{ mio_badge | safe }}" alt="mio" /></td>
-		</tr>
-
-		<tr>
-			<td>
-			Version 0.4.4: <a href="{{ mio_badge | safe }}?version=0.4.4">{{ mio_badge }}?version=0.4.4</a>
-			</td>
-			<td><img src="{{ mio_badge | safe }}?version=0.4.4" alt="mio" /></td>
-		</tr>
-
-		<tr>
-			<td>
-			Version 0.1.0: <a href="{{ mio_badge | safe }}?version=0.1.0">{{ mio_badge }}?version=0.1.0</a>
-			</td>
-			<td><img src="{{ mio_badge | safe }}?version=0.1.0" alt="mio" /></td>
-		</tr>
-		</tbody>
-	</table>
+	<div class="purge-g">
```

Comment:
Is this class documented somewhere? I don't remember it being a custom docs.rs class.

Reply:
It's a pure-css grid class: https://purecss.io/grids/

Reply:
I think you meant to add `pure-g` then, `purge-g` doesn't have any results. Which makes me think this class isn't actually doing anything - does the page look any different if you remove it?

Reply:
🤦 reading is very difficult ^^ the only thing needed is the div.